### DISCUSSION TITLE
🙏 QCefView支持Scheme么

### DISCUSSION THREAD

**Discussion by orgcrab** [CefView/QCefView#182] [Q&A] [answered]
CefViewCore中有看到scheme相关的实现，但在QCefView中没有与之对应的接口

**Comment by tishion** [comment, marked as answer]
很久以前是支持的，但是当时QCefView的封装接口我设计的不太优雅，所以就直接移除了。不过custom scheme确实是一个很有用的功能，所以后面考虑再加回来。
